docs/source/atomdb_cite.bib

```diff
@@ -0,0 +1,8 @@
+@Article{atomdb,
+    author = {S{\'a}nchez D{\'i}az, Gabriela and Richer, Michelle and Tehrani, Alireza and 
```

Comment:
author = {S{\'a}nchez D{\'i}az, Gabriela and Richer, Michelle and Mart\'{i}nez Gonz\'{a}lez, Marco and van Zyl, Maximilian and Pujal, Leila and Tehrani, Alireza and Bianchi, Julianna and Ayers, Paul W. and Heidar-Zadeh, Farnaz},

Reply:
Done!